### PR TITLE
removed anchors and labels from cockpit

### PR DESCRIPTION
**Pull request by rssag**

### CODE REVIEW

File: content/cockpit/widgets-collection.md [hunk 1]
```diff
@@ -77,13 +69,12 @@ The "Asset properties" widget displays a user-defined list of attributes of the
...
 |Target assets or devices|Select groups or devices.
-|Properties|List of properties, see [Asset table](#widget-asset-table).
+|Properties|List of properties, see [Asset table](#asset-table).
 
 {{< c8y-admon-info >}}
 In the view mode, this widget only displays the properties which are not empty.
 {{< /c8y-admon-info >}}
 
-<a name="widget-asset-table"></a>
 ### Asset table
```
Comment (BeateRixen) [inline content/cockpit/widgets-collection.md:78]: We have quite some links to the various widgets throughout the documentation. If we remove the anchors here, we should either add ids if the names do not match the headings or search for them throughout the docs and change them.

Comment (rssag) [reply content/cockpit/widgets-collection.md:78]: i did a search in `/content` for every removed anchor.
there should be no links to them in the documentation

**Comment by rssag** [comment]
removed the labels added for edge, and edited edge instead

File: content/cockpit/managing-exports.md [hunk 2]
```diff
@@ -29,7 +29,10 @@ ROLES & PERMISSIONS:
 
 With this feature, you can request data for the whole tenant. Additionally, you can choose to filter for specific devices, time ranges or fields. The export data contains information about all specified filters and enabled fields.
 
-<a name=""></a>
+{{< c8y-admon-related >}}
```
Comment (MWindel) [inline content/cockpit/managing-exports.md:32]:
```suggestion

```

File: content/cockpit/managing-exports.md [hunk 3]
```diff
@@ -29,7 +29,10 @@ ROLES & PERMISSIONS:
...
+- [Analytics > Streaming Analytics](/streaming-analytics/overview-analytics/) for details on customizing scheduled exports using the Streaming Analytics application.
```
Comment (MWindel) [inline content/cockpit/managing-exports.md:33]:
```suggestion

```

**Review by MWindel — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**